resources/prometheus/prometheus-rules.yaml

```diff
@@ -1013,3 +1013,42 @@ spec:
           for: "7d"
           labels:
             severity: "warning"
+    - name: secured-cluster.alerts
+      rules:
+        - expr: |
+            (max by (namespace)(rox_central_secured_clusters) /  max by (namespace)(max_over_time(rox_central_secured_clusters[72h])) < bool 0.8)
+            and on (namespace) (avg_over_time (rox_central_secured_clusters[1w]) >= 5)
+          record: rhacs_tenants:namespace:central:rhacs_secured_cluster_drop
+        - alert: TenantSecuredClusterDrop
+          expr: |
+            rhacs_tenants:namespace:central:rhacs_secured_cluster_drop >= 1
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: Tenant '{{ $labels.namespace }}' lost at least 20% of its secured clusters
+            description: Tenant '{{ $labels.namespace }}' lost at least 20% of its secured clusters
+            sop_url: "TODO: SoP URL"
+        - expr: |
+              sum(rox_central_secured_clusters) / max_over_time(sum(rox_central_secured_clusters)[7d:])
```

Comment:
I'm not familiar enough with the prometheus syntax so just double checking, why is the max_over_time above [72h] but here [7d:] - does the sum affect the range specification?

Reply:
No reasons changed it to 7d: for every query. Now its 7d for everything.

Reply:
Actually that's wrong there is a diff and I did not understand it. [7d:] would yield a vector where the query is evaluated on every single timestamp of a sample rox_central_secured_clusters whereas [7d] would only evaluate once for the current point in time.